src/app/body/create-new-session/create-new-session.component.ts

```diff
@@ -29,7 +29,7 @@ export class CreateNewSessionComponent implements OnInit {
   todayDate: string
   description: string
   assigneeName: string
-  creatorName: string
+  creatorName = this.authService.user.email
```

Comment:
Add the "string" specification. On ngOnInit you are calling the getUserEmail() function in auth service. So as explained before, do not assign value before the constructor